src/components/HelpScoutBeacon/HelpScoutBeacon.js

```diff
@@ -444,80 +508,11 @@ const Paragraph = styled.p`
   margin-top: ${1 * GU}px;
   font-size: 15px;
   line-height: 26px;
-
-  ${breakpoint(
-    'medium',
-    `
-      font-size: 13px;
-      line-height: 21px;
+  ${({ isSmall }) =>
+    isSmall &&
     `
-  )}
-`
-
-const Wrapper = styled.aside`
-  background: ${({ theme }) => theme.helpSurface};
-  position: absolute;
-  bottom: ${-2 * GU}px;
-  right: ${-2 * GU}px;
-  width: 100vw;
-  height: 100vh;
-  z-index: 1;
-  overflow: hidden;
-  display: flex;
-  flex-direction: column;
-
-  ${breakpoint(
-    'medium',
-    `
-      bottom: calc(-40px - ${4 * GU}px);
-      width: 336px;
-      height: 482px;
-      position: unset;
-      box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.15);
-      border-radius: ${RADIUS}px;
-    `
-  )}
-`
-
-const Header = styled.header`
-  position: relative;
-  height: 240px;
-  background-color: ${({ theme }) => theme.help};
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  flex-direction: column;
-
-  ${breakpoint(
-    'medium',
-    `
-      height: 148px;
-      /* needs both height and min-height as button uses flex: 1
-       * and would push this upwards */
-      min-height: 148px;
-      border-top-right-radius: ${RADIUS}px;
-      border-top-left-radius: ${RADIUS}px;
-    `
-  )}
-`
-
-const HeaderImage = styled.img`
-  width: 300px;
-  height: 139px;
-  position: absolute;
-  bottom: -6px;
-`
-
-const Main = styled.main`
-  height: 100%;
-  display: flex;
-  flex-direction: column;
-  padding: ${5 * GU}px ${3 * GU}px ${3 * GU}px;
-  color: ${({ theme }) => theme.helpSurfaceContent};
-`
-
-const Heading = styled.h3`
-  ${textStyle('title4')};
+        font-size: 13px; line-height: 21px;
```

Comment:
```suggestion
        font-size: 13px;
        line-height: 1.6;
```

Reply:
Also moved this component to be a functional component, to avoid what was happening in `NotificationModals`.